### PR TITLE
Adding changes for issue 319

### LINKED ISSUE #319: Rho man page: edits part 1, name/synopsys/description/usage intro sections
Man page changes, part 1 
These changes go from the beginning of the document through the intro of the Usage section (before the Authentication section starts). Other issues will cover changes to the subsequent sections.

### PR DESCRIPTION
**Pull request by jlprevatt-zz**
Submitting changes for the issue 319 "part 1" edit of the man page.

Closes #319

### CODE REVIEW

File: doc/source/man.rst [hunk 1]
```diff
@@ -15,36 +15,38 @@ Synopsis
 Description
 -----------
 
-rho is a network discovery tool to identify the number of systems running on a network, their operating system and versions of some key packages and products for almost any Linux or Unix flavor. Being able to identify the systems running on the network is a vital component to managing entitlements (licenses and renewals). Ultimately, discovery is part of the larger sysadmin task of managing inventories.
+Rho, and the rho command, is a network discovery tool to identify environment data, or *facts*, such as the number of physical and virtual systems on a network, their operating systems and other configuration data, and versions of some key packages and products for almost any Linux or UNIX version. The ability to discover and identify the software and systems that are running on the network improves your ability to understand and report on your entitlement usage. Ultimately, this discovery is part of the larger system administration task of managing your inventories.
```
Comment (chambridge) [inline doc/source/man.rst]: perhaps:

> ... is a network discovery and inspection tool ...


and 

> ... Ultimately, this discovery and inspection is part of the larger ...

I feel like discovery is just locating and connecting to the system and the inspection part is where we find out about the system.

**Review by noahl — APPROVED** [review]
This is a major improvement to our man page. Thanks, @jlprevatt !

**Review by chambridge — COMMENTED** [review]
Looks fine. Made a comment about the usage of only discovery, where as I think about it as discovery and inspection, but I'm flexible on this.  Discovery is used on line 18, 20, and 24 if you wanted to make consistent updates.

**Comment by jlprevatt-zz** [comment]
Please make sure that the short code strings at the beginning of the Usage section are okay. 
Remember, this is a short edit that can stop at the header for the Authentication section. Those changes will come in the next batch.

**Review by chambridge — APPROVED** [review]
Great!

**Review by kdelee — APPROVED** [review]
LGTM 👍 

thanks!

**Review by elyezer — APPROVED** [review]
Thank you, it looks good.

**Pull request merged**